src/Sylius/Bundle/CoreBundle/DependencyInjection/Compiler/SyliusLegacyAliasesPass.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+final class SyliusLegacyAliasesPass implements CompilerPassInterface
+{
+    public const PLUGIN_TO_SYLIUS_PREFIXES_MAP = [
+        'Sylius\PriceHistoryPlugin\Application\Calculator\ProductVariantLowestPriceCalculator' => 'Sylius\Component\Core\Calculator\ProductVariantPriceCalculator',
+        'Sylius\PriceHistoryPlugin\Application\Checker' => 'Sylius\Component\Core\Checker',
```

Comment:
Similar to the comment above: https://github.com/Sylius/Sylius/pull/14911/files#r1159464601